srcalc/src/main/java/gov/va/med/srcalc/db/VariableDao.java

```diff
@@ -48,14 +48,21 @@ protected Session getCurrentSession()
     }
     
     /**
-     * Returns the persistent Variable with the given display name, or null
+     * Returns the persistent Variable with the given key, or null
      * if no such Variable exists.
      */
-    public AbstractVariable getByName(final String displayName)
+    public AbstractVariable getByKey(final String variableKey)
     {
         final Query q =  getCurrentSession().createQuery(
+                "from AbstractVariable v where v.key = :variableKey");
+        q.setString("variableKey", variableKey);
+        return (AbstractVariable)q.uniqueResult();
+    }
+
+	public AbstractVariable getByDisplayName(final String displayName) {
```

Comment:
Is this method used anymore? I don't think there's any reason to lookup a variable by display name now that we have a unique key.